lib/src/main/java/com/alphawallet/token/entity/AttributeType.java

```diff
@@ -234,6 +247,30 @@ else if (data.startsWith("0x"))
         }
     }
 
+    //Sometimes value needs to be processed from the raw input.
+    //Currently only time
+    public BigInteger processValue(BigInteger val) throws UnsupportedEncodingException, ParseException
+    {
+        switch (syntax)
+        {
+            case GeneralizedTime:
```

Comment:
@JamesSmartCell want to add the missing cases? if not implemented we can still put them in and simply break

Reply:
There's no missing cases. Currently we only need to transform time attribute value (ie convert the string representation to Unix time value). In future there may be other attributes that need transforming, but currently we don't know what they are.

Reply:
Added the missing but not used cases like Boolean, BitString etc. While these are not being implemented, the switch should be exhaustive otherwise it would be better to just use an if statement